recipes/isitgr/meta.yaml

```diff
@@ -0,0 +1,63 @@
+{% set name = "isitgr" %}
+{% set version = "1.0.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/mishakb/ISiTGR/archive/{{ version }}.tar.gz
+  sha256: ebf53b46ce3682a0154f3e713e8d320f5f7588fe1cb7a6fe5c9db8d1cad78b59
+
+build:
+  number: 0
+  skip: True  # [py2k]
+  skip: True  # [win]
+
+requirements:
+  build:
+    - {{ compiler('c') }}  # [not win]
+    - {{ compiler('fortran') }}  # [not win]
+#    - m2w64-toolchain  # [win]
+    - git  # [not win]
+#    - m2-git  # [win]
+    - make  # [not win]
+#    - m2-make  # [win]
+    - python
+    - setuptools
+    - pip
+    - llvm-openmp  # [osx]
```

Comment:
Should this and `libgomp` be in `host`? Or is there something special about open MP?

Reply:
build is correct: https://conda-forge.org/docs/maintainer/knowledge_base.html#openmp

it is part of the compiler